charts/datadog/templates/container-agent.yaml

```diff
@@ -83,6 +83,14 @@
 {{ toYaml .Values.agents.containers.agent.env | indent 4 }}
 {{- end }}
   volumeMounts:
+    - name: installinfo
+      subPath: install_info
+      {{- if eq .Values.targetSystem "windows" }}
+      mountPath: C:/datadog-agent/install_info
```

Comment:
I think default path on Windows is `c:\programdata\datadog\`

Reply:
Good catch! Fixed!